docs/prerequisites.md

```diff
@@ -0,0 +1,44 @@
+# Prerequisites
```

Comment:
should we link to this document from README.md in the root folder?

Reply:
Yes. Good idea!